### ISSUE TITLE
Idea/ proposal form: Missing tab focus highlights on image upload

### ISSUE THREAD

**Issue by sblindow** [liqd/a4-meinberlin#6289]
**URL:** 

Idea/Proposal form

**user:** 

Authenticated keyboard users

**expected behaviour:** 

Tab focus highlights should appear on all interactive elements during keyboard navigation.

**behaviour:** 

Image upload controls lack visible tab focus highlights.
